config/environments/development.rb

```diff
@@ -38,4 +38,10 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Prevent "Blocked host: hostname"
+  if !ENV["APP_HOSTNAME"].nil?
```

Comment:
Could you use '.present?' here instead maybe?

Reply:
Maybe! I'll have to defer to you as the Ruby expert in this case. I'll give it a go and see if it still works afterward!

I've also got some tweaks to make to the GHA config based on [some advice from @crazy-max](https://github.com/docker/build-push-action/pull/426#issuecomment-932776148)

Reply:
`.present?` didn't work. I had thought about using `!ENV["APP_HOSTNAME"].nil? && !ENV["APP_HOSTNAME"].empty?`, but I don't know if that's any better

Reply:
Ah wait, I had the logic reversed... one sec